### ISSUE TITLE
table to visual

### ISSUE THREAD

**Issue by karissa** [paulfitz/daff#24]
I wrote this thing to convert a couple daff tables to a visual for convenience. How do you feel about putting it somewhere in daff core (or maybe extend it even to a version that is smart about 3-table diffs, too). 

If not, I'll open an npm module for it

https://github.com/karissa/dat-visualdiff/blob/master/lib/dat2daff.js#L33
```js
function tablesToVisual (tables, opts) {
  var flags = new daff.CompareFlags();

  var table1 = tables[0]
  var table2 = tables[1]

  var alignment = daff.compareTables(table1, table2, flags).align();
  var highlighter = new daff.TableDiff(alignment,flags);
  var table_diff = new daff.SimpleTable();
  highlighter.hilite(table_diff);

  if (opts.html) {
    var diff2html = new daff.DiffRender();
    diff2html.render(table_diff);
    var table_diff_html = diff2html.html();
    return table_diff_html
  }
  else {
    var diff2terminal = new daff.TerminalDiffRender();
    return diff2terminal.render(table_diff)
  }
}
```

**Comment by paulfitz** [comment]
I'm definitely +1 on adding a helper that does this in core. Even I have a hard time remembering all the hoops to jump through to make a diff...

**Comment by karissa** [comment]
Hehe ok! cool

**Comment by paulfitz** [comment]
this became #26 which was merged as 3bc43102f5d377cec915f6413cb525b192775a4e and a4e1d2bb536ea0835462ef670304790e0969b476, in daff 1.2.13. Thanks @karissa.
